packages/hash/frontend/src/components/layout/PageHeader/PageHeader.tsx

```diff
@@ -1,25 +1,40 @@
-import Link from "next/link";
 import React from "react";
 import { useModal } from "react-modal-hook";
-import { tw } from "twind";
+import { Box, Button, useTheme, useMediaQuery } from "@mui/material";
+import { useRouter } from "next/router";
 
-import { Button } from "../../forms/Button";
 import { useLogout } from "../../hooks/useLogout";
 import { useUser } from "../../hooks/useUser";
 import { LoginModal } from "../../Modals/AuthModal/LoginModal";
 import { AccountDropdown } from "./AccountDropdown";
 import { SearchBar } from "./SearchBar";
-import { SIDEBAR_WIDTH } from "../PageSidebar/PageSidebar";
+import { HashNavIcon } from "../../icons";
+import { Link } from "../../Link";
+import { ActionsDropdown } from "./ActionsDropdown";
+import { NotificationsDropdown } from "./NotificationsDropdown";
 
-interface NavProps {
-  tw?: string;
-}
-
-const Nav: React.FC<NavProps> = ({ tw: tw_, children }) => (
-  <nav className={tw`container mx-auto flex ${tw_}`}>{children}</nav>
+const Nav: React.FC = ({ children }) => (
+  <Box
+    component="nav"
+    sx={{
+      width: "100%",
+      display: "flex",
+      justifyContent: "space-between",
+      px: { xs: 2, md: 3 },
+    }}
+  >
+    {children}
+  </Box>
 );
 
 export const PageHeader: React.VFC = () => {
+  const theme = useTheme();
+  const router = useRouter();
+
+  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
+
+  const { accountId } = router.query as Record<string, string>;
```

Comment:
This assumes `accountId` will always be provided in the router query, which might be the case right now because most pages live under`[accountId]/...`, but don't think this is an assumption we should make in this component. I think it'd be more appropriate to make this a prop of `PageHeader` instead.